contrib/devtools/clog/main.go

```diff
@@ -141,36 +119,43 @@ func main() {
 		verboseLog.SetPrefix(logPrefix)
 	}
 
+	config = mustFindAndReadConfig(checkGetcwd())
 	if err := RootCmd.Execute(); err != nil {
 		log.Fatal(err)
 	}
 }
 
 func addEntryFileFromConsoleInput() error {
-
 	reader := bufio.NewReader(os.Stdin)
-	fmt.Println("Please enter the section (either: \"breaking\", \"features\", \"improvements\", \"bugfixes\")")
+	fmt.Fprintf(os.Stderr, "Please enter the section %v: ", config.Sections)
 	sectionDir, _ := reader.ReadString('\n')
 	sectionDir = strings.TrimSpace(sectionDir)
-	if _, ok := sections[sectionDir]; !ok {
+	if _, ok := config.Sections[sectionDir]; !ok {
 		return errors.New("invalid section, please try again")
 	}
 
-	fmt.Println("Please enter the stanza (either: \"gaia\", \"gaiacli\", \"gaiarest\", \"sdk\", \"tendermint\")")
-	stanzaDir, _ := reader.ReadString('\n')
-	stanzaDir = strings.TrimSpace(stanzaDir)
-	if _, ok := stanzas[stanzaDir]; !ok {
-		return errors.New("invalid stanza, please try again")
+	fmt.Fprintf(os.Stderr, "Please enter the tag %v: ", config.Tags)
```

Comment:
Shouldn't I be able to enter multiple tags?

e.g.
```
(tendermint) (gaia) #1234 My log message.
```

Reply:
Oh that's interesting. Yes, I think we should allow multiple tags to avoid creating multiple entry files for the same issue/PR. Will redesign